x-pack/test/security_solution_api_integration/test_suites/detections_response/basic_essentials_license/rule_management/coverage_overview.ts

```diff
@@ -0,0 +1,585 @@
+            supertest,
+            log,
+            getCustomQueryRuleParams({ rule_id: 'rule-2', index: ['index-pattern-2'] })
+          );
+
+          const body = await getCoverageOverview(supertest, {
+            search_term: 'index-pattern-2',
+          });
+
+          expect(body).to.eql({
+            coverage: {},
+            unmapped_rule_ids: [expectedRule.id],
+            rules_data: {
+              [expectedRule.id]: {
+                activity: 'disabled',
+                name: 'Custom query rule',
+              },
+            },
+          });
+        });
+      });
+
+      describe('activity', () => {
+        it('returns response filtered by disabled rules', async () => {
+          const expectedRule = await createRule(
+            supertest,
+            log,
+            getCustomQueryRuleParams({ rule_id: 'rule-1', threat: generateThreatArray(1) })
+          );
+          await createRule(
+            supertest,
+            log,
+            getCustomQueryRuleParams({
+              rule_id: 'rule-2',
+              enabled: true,
+              threat: generateThreatArray(2),
+            })
+          );
+
+          const body = await getCoverageOverview(supertest, {
+            activity: [CoverageOverviewRuleActivity.Disabled],
+          });
+
+          expect(body).to.eql({
+            coverage: {
+              T001: [expectedRule.id],
+              TA001: [expectedRule.id],
+              'T001.001': [expectedRule.id],
+            },
+            unmapped_rule_ids: [],
+            rules_data: {
+              [expectedRule.id]: {
+                activity: 'disabled',
+                name: 'Custom query rule',
+              },
+            },
+          });
+        });
+
+        it('returns response filtered by enabled rules', async () => {
+          await createRule(
+            supertest,
+            log,
+            getCustomQueryRuleParams({ rule_id: 'rule-1', threat: generateThreatArray(1) })
+          );
+          const expectedRule = await createRule(
+            supertest,
+            log,
+            getCustomQueryRuleParams({
+              rule_id: 'rule-2',
+              enabled: true,
+              threat: generateThreatArray(2),
+            })
+          );
+
+          const body = await getCoverageOverview(supertest, {
+            activity: [CoverageOverviewRuleActivity.Enabled],
+          });
+
+          expect(body).to.eql({
+            coverage: {
+              T002: [expectedRule.id],
+              TA002: [expectedRule.id],
+              'T002.002': [expectedRule.id],
+            },
+            unmapped_rule_ids: [],
+            rules_data: {
+              [expectedRule.id]: {
+                activity: 'enabled',
+                name: 'Custom query rule',
+              },
+            },
+          });
+        });
+
+        it('returns all rules if both enabled and disabled filters are specified in the request', async () => {
```

Comment:
Let's also add a test `returns all rules if neither of enabled and disabled filters are specified in the request`